pkgs/development/python-modules/latex2pydata/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, setuptools
+, nix-update-script
+}:
+
+buildPythonPackage rec {
+  pname = "latex2pydata";
+  version = "0.2.0";
+
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-lFYGBFox7fv/vlfqZN3xsh9UIRCQ+C5Cizq9j4RTcJ0=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
I'm not sure this has any value if updating this package is trivial with `nix-update` or `nixpkgs-update`.

Reply:
I think r-ryantm uses the update script to try upgrading periodically.

Reply:
> I think r-ryantm uses the update script to try upgrading periodically.

I don't think it will change the schedule of the upgrade attempts just because of this attribute. Try to find evidence for that in these projects' documentations, and if you don't find any, please remove it.

Reply:
I have removed it.